### PR TITLE
Documentation updates, rename common -> setup

### PR DESCRIPTION
**Pull request by makeworld-the-better-one**
@YurkoWasHere I've put a `TODO` in several places where more documentation is needed but I don't have the knowledge. I was hoping you could fill in those sections, then I can edit if needed, before merging the PR.

I have not yet tested these modules. No substantial code changes have been made, so fixes from testing can be in a different PR.

:exclamation: I've updated the version number in the galaxy file, because this PR contains a breaking change. :exclamation:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,19 +15,23 @@ ansible-galaxy install -r requirements.yml
...
-- [prometheus_alertmanager](roles/prometheus_alertmanager/README.md)
-- [prometheus_exporter_addnode](roles/prometheus_exporter_addnode/README.md)
-- [prometheus_exporter_docker](roles/prometheus_exporter_docker/README.md)
-- [provision_ssh_keys](roles/provision_ssh_keys/README.md)
-- [proxmox_provision_vm](roles/proxmox_provision_vm/README.md) - Provision VM on ProxMox
-- [proxmox_server](roles/proxmox_server/README.md) - setups ProxMox server for provisioning
-- [ssl](roles/ssl/README.md) - Parent to NGINX, provdes ssl certificates
-- [unattended_upgrades](roles/unattended_upgrades/README.md) - Unatteded updates
\ No newline at end of file
+- [hypha.common.setup](roles/setup/README.md) - setup server for use
```
Comment (darkdrgn2k) [inline README.md:18]:
```suggestion
- [setup](roles/setup/README.md) - setup server for use
```
Probably want to conform to the other entries

Comment (makeworld-the-better-one) [reply README.md:18]: Yeah I should be more clear about what I mean. My point was that `setup` alone conflicts with the ansible built-in. In general we should probably be using full paths for everything, ie `ansible.builtin.setup`. Will fix.

**Comment by makeworld-the-better-one** [comment]
I've updated the version number in the galaxy file, because this PR contains a breaking change. Maybe this PR should go into a dev branch or something where many changes can be aggregated before making a 1.0.0 release, instead of this small documentation update getting a new version? To make all the breaking changes at once. :shrug:

**Comment by makeworld-the-better-one** [comment]
This PR is waiting on #17

**Comment by makeworld-the-better-one** [comment]
Change base branch to `dev` so we don't to worry about breaking changes, updating versions, etc.

**Pull request merged**